src/Sylius/Bundle/ApiBundle/Command/CreateClientCommand.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command;
+
+use FOS\OAuthServerBundle\Model\ClientManagerInterface;
+use Sylius\Bundle\AdminApiBundle\Model\Client;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class CreateClientCommand extends Command
+{
+    /** @var ClientManagerInterface|null */
+    private $clientManager;
+
+    public function __construct(?string $name = null, ClientManagerInterface $clientManager = null)
```

Comment:
We should not allow to `null` the `$clientManager`. Moreover, we could take only this argument and pass the command name in `parent::__construct('name')` or (as it's done now) in the `configure()` method

Reply:
```suggestion
    public function __construct(ClientManagerInterface $clientManager)
```